### ISSUE TITLE
If checkpoints are available

### ISSUE THREAD

**Issue by yangsizhe** [liruiw/GenSim#3]
Glad to see a wonderful work!
I wonder if there are checkpoints available. I used the checkpoints in the original cliport repo and found that the success rate is very low.

**Comment by liruiw** [comment]
You mean you evaluate the cliport pretrained checkpoint on the GPT generated tasks and get a low success rates? Which environments do you test on, and what are the success rates?

**Comment by yangsizhe** [comment]
Not on the GPT generated tasks. I test on the stack-block-pyramid-seq-seen-colors task, and the success rate is 0. I evaluate the same checkpoint using the code in the original cliport repo (https://github.com/cliport/cliport), and the success rate is 1. I notice that you modify the original cliport in order to support batchsize>1. I wonder if the original cliport (https://github.com/cliport/cliport) pretrained checkpoint can be used here.

**Comment by liruiw** [comment]
Interesting. i have not seen this problem before. Did you notice anything unusual when you load the state_dict? Can you visualize the affordance using the cliport notebook?

**Comment by yangsizhe** [comment]
I did not notice anything unusual when I load the state_dict. The affordance maps are in the notebook (https://colab.research.google.com/drive/1cXViRgIHMv73rYjNgCFSo00yF1QeDgaK?usp=sharing). It seems that the agent has wrong choice of color when I evaluate the same checkpoint using the code in this repo.

**Comment by liruiw** [comment]
I cannot seem to view the affordance map. This seems to indicate a bug, and I would like to investigate more. Have you tried other environments btw? What about simpler ones like align_box_corner?

**Comment by yangsizhe** [comment]
I tried align_box_corner just now. The affordance map is also incorrect. The pick affordance indicates the agent should pick up the green corner. I'm not sure if my carelessness caused the problem, have you ever tried the checkpoints in the original cliport repo (https://github.com/cliport/cliport) for you code?

**Comment by liruiw** [comment]
I've been able to train a single-task model and evaluate on my codebase. I only tried the cliport checkpoint a while ago. I think there are potential changes on padding, rotations, x/y order that might cause this. Feel free to debug further. I will be able to take a closer look sometime later as well.

**Comment by yangsizhe** [comment]
OK, thank you!

**Comment by liruiw** [comment]
Turns out this problem is due to the seen/unseen color split problem [here](https://github.com/liruiw/GenSim/blob/main/cliport/utils/utils.py#L1060). The trained model on cliport is evaluated on a different set of colors but this repo does not consider that kind of color-level generalizations. One way to solve it is to always use the seen color split.
